recipes/troposphere/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "troposphere" %}
+{% set version = "2.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/cloudtools/{{ name }}/archive/{{ version }}.tar.gz"
+  sha256: "75ec21c7c9792a637f8370a71849e0d336f43014eee5949e698aa1bd6587b211"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - cfn_flip >=1.0.2
```

Comment:
In conda forge, its **`cfn-flip`** rather than `cfn_flip`. I installed just a minute ago. will that affect the way its called in troposphere?

Reply:
just looked at it: troposphere calls it as  `cfn_flip`.
     

https://github.com/cloudtools/troposphere/blob/e7cb5842f3ba1829a72be2ecb7ce140bbc370592/troposphere/__init__.py#L7-L9

we might need to resubmit the the feedstock.

Reply:
Good catch. It's also missing awacs, which I'll add. And awacs requires slimit, so I gotta add that too.